p2p/bootstrap.go

```diff
@@ -43,37 +43,57 @@ func (m *p2pModule) configureBootstrapNodes() error {
 
 // bootstrap attempts to bootstrap from a bootstrap node
 func (m *p2pModule) bootstrap() error {
-	var pstore typesP2P.Peerstore
+	var wg sync.WaitGroup
 
-	for _, bootstrapNode := range m.bootstrapNodes {
-		m.logger.Info().Str("endpoint", bootstrapNode).Msg("Attempting to bootstrap from bootstrap node")
+	for _, serviceURL := range m.bootstrapNodes {
+		wg.Add(1)
```

Comment:
Where does Done get called for these added items? I don't see what will let the Wait call below pass priority.

Reply:
:thinking: great catch! I'm honestly questioning how this is working now. I must have made an invalid assumption at some point. Lemme look deeper. :sweat_smile: